localstack/utils/container_utils/container_client.py

```diff
@@ -974,3 +980,16 @@ def resolve_dockerfile_path(dockerfile_path: str) -> str:
         if os.path.isdir(dockerfile_path) and os.path.exists(rel_path):
             return rel_path
         return dockerfile_path
+
+
+def get_docker_client() -> ContainerClient:
```

Comment:
Is there a reason for explicitely using this instead of DOCKER_CLIENT (which uses docker sdk inside the container, if installed, and the cmd otherwise?) as created by create_docker_client in docker_utils.py?

Reply:
Great catch, thanks! Of course, much better to use the existing one - updated the PR 👍